### ISSUE TITLE
remove_static_symbol error

### ISSUE THREAD

**Issue by GeT1t** [lief-project/LIEF#321]
**Describe the bug**
remove_static_symbol  lief.not_found: Can't find ' ' it

**To Reproduce**

`import lief

library = lief.parse("libcompatible.so")

for symbol in library.exported_symbols:
    if symbol.name.startswith("\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n"):
        library.remove_static_symbol(symbol)
        
library.write("libcompatible.new.so")`

**Expected behavior**

**Environment (please complete the following information):**
 - System and Version : Windows 7 x64
 - Target format : ELF
 - LIEF commit version: 0.10.0-826965b

**Comment by romainthomas** [comment]
Indeed, I didn't think about such case. Do you have a sample with this kind of symbol ?

**Comment by GeT1t** [comment]
[libcompatible.zip](https://github.com/lief-project/LIEF/files/3455060/libcompatible.zip)

**Comment by GeT1t** [comment]
anyone ready to solve this problem?

**Comment by GeT1t** [comment]
@romainthomas some suggestion？

**Comment by romainthomas** [comment]
This is now fixed
